webapp/src/main/resources/properties/zh-Hans.properties

```diff
@@ -0,0 +1,423 @@
+# Label displayed in the header menu to open the workbench
+header.workbench=Workbench
+
+# Label displayed in the header menu to open the repositories page
+header.repositories=Repositories
+
+# Label displayed in the header menu to open the project requests page
+header.projectRequests=Project Requests
+
+# Label displayed in the header menu to open the settings page
+header.settings=Settings
```

Comment:
@jeeyi we never got the translation for this? just wondering :)

Reply:
Hanna was working with some students to translate them. I need to look at the old email threads on it.